packages/sql-contract/tests/AGENTS.md

```
@@ -137,3 +137,16 @@ Each test must:
 
 When in doubt:
 Make the test stricter, not the mapper looser.
+
+--- 
```

markdownlint-cli2 (0.18.1)

Horizontal rule style
Expected: ---; Actual: --- 

(MD035, hr-style)